src/core/trulens/core/database/sqlalchemy.py

```diff
@@ -973,6 +974,16 @@ def get_datasets(self) -> pd.DataFrame:
                 columns=["dataset_id", "name", "meta"],
             )
 
+    def insert_event(self, event: Event) -> types_schema.EventID:
+        """See [DB.insert_event][trulens.core.database.base.DB.insert_event]."""
+        with self.session.begin() as session:
+            _event = self.orm.Event.parse(event, redact_keys=self.redact_keys)
+            session.add(_event)
```

Comment:
does `add` just fill in the `event_id` or something and that's why we needed it?

Reply:
oh this does the addition to the database

Reply:
`event_id` was needed more because of sqlalchemy - maybe there's a way to remove that requirement, not sure how to do it

Reply:
it's not a big deal so don't tell me if it takes more than like 2 minutes to figure out, but what was the error when you didn't have this column?